airflow/providers/mongo/hooks/mongo.py

```diff
@@ -20,10 +20,12 @@
 
 from ssl import CERT_NONE
 from types import TracebackType
+from typing import Any, overload
 from urllib.parse import quote_plus, urlunsplit
 
 import pymongo
 from pymongo import MongoClient, ReplaceOne
+from typing_extensions import Literal
```

Comment:
Should `airflow.typing_compat` be used here?

Reply:
of course. Silly me.